src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -2537,7 +2534,7 @@ public SatisfactionRating createSatisfactionRating(
 
   public Iterable<DynamicContentItem> getDynamicContentItems() {
     return new PagedIterable<>(
-        cnst("/dynamic_content/items.json"), handleList(DynamicContentItem.class, "items"));
+        cbp("/dynamic_content/items.json"), handleList(DynamicContentItem.class, "items"));
```

Comment:
Not in the list provided in [support.zendesk.com/hc/en-us/articles/5591904358938-New-limits-for-offset-based-pagination#h_01GNCT8Z7DJ7G4TQ3CFFXQ007Z](https://support.zendesk.com/hc/en-us/articles/5591904358938-New-limits-for-offset-based-pagination#h_01GNCT8Z7DJ7G4TQ3CFFXQ007Z)

The endpoint documentation doesn't include CBP either [developer.zendesk.com/api-reference/ticketing/ticket-management/dynamic_content/#list-items](https://developer.zendesk.com/api-reference/ticketing/ticket-management/dynamic_content/#list-items)